packages/mobile/e2e/src/usecases/ExchangeCelo.js

```diff
@@ -143,19 +198,29 @@ export default ExchangeCelo = () => {
     await element(by.id('ConfirmWithdrawButton')).tap()
     // Enter PIN
     await enterPinUiIfNecessary()
+    // Get transaction time
+    let transactionTime = new Date()
     // Scroll to top of feed
     await element(by.id('ExchangeScrollView')).scroll(200, 'down')
     // Wait 10 seconds checking that error banner is not visible each second
     await waitForExpectNotVisible('errorBanner')
-    // Assert balance is updated
-    await waitFor(element(by.id('CeloBalance')))
-      .toHaveText(`${(+celoBalanceBefore.text - CELO_TO_WITHDRAW).toFixed(3)}`)
-      .withTimeout(10 * 1000)
-    // Assert withdrawal amount is present
-    await expect(
+    // Wait up to 30 second and assert the transaction correctly appears
+    await waitFor(
       element(
         by.text(`-${CELO_TO_WITHDRAW.toFixed(3)}`).withAncestor(by.id('TransactionList'))
       ).atIndex(0)
-    ).toBeVisible()
+    )
+      .toBeVisible()
+      .withTimeout(15 * 10000)
```

Comment:
This is 15 * 10000, I think there's an extra 0?

Should we put this timeout in a constant at the top? And how would you feel about increasing it a bit, sometimes it can be a bit slow

Reply:
There was an extra zero thanks! I use different numbers in the spec so a constant doesn't make sense.